apps/userdashboard/assets/js/a4_candy_userdashboard/react_moderation_comment_list.jsx

```diff
@@ -0,0 +1,18 @@
+import ModerationCommentList from './ModerationCommentList'
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+function init () {
+  $('[data-aplus-widget="moderation_comment_list"').each(function (i, element) {
```

Comment:
using vanilla JS to select the <div data-aplus-widget> instead of jQuery.
Something like that:

.querySelectorAll('div[data-aplus-widget]')

Reply:
missing bracket :)